Description: replace ava by tape
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-10-06

--- a/test.js
+++ b/test.js
@@ -1,6 +1,6 @@
-import path from 'path';
-import test from 'ava';
-import resolvePkg from '.';
+const path = require('path');
+const test = require('tape');
+const resolvePkg = require('.');
 
 test('main', t => {
 	t.is(resolvePkg('nonexistent'), undefined);
@@ -9,4 +9,5 @@
 	t.is(path.relative('.', resolvePkg('grunt-svgmin/tasks')), 'node_modules/grunt-svgmin/tasks');
 	t.is(path.relative('.', resolvePkg('@someprivate/module-test')), 'node_modules/@someprivate/module-test');
 	t.is(path.relative('.', resolvePkg('@someprivate/module-test/subdir')), 'node_modules/@someprivate/module-test/subdir');
+    t.end();
 });
